IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcTranslationOperation/Documentation.md

```diff
@@ -1 +1 @@
-A translation operation specifies an offset in the coordinate reference system. It does not specify any conversion or distortions only a translation.
+A translation operation specifies an offset in the coordinate reference system. It does not specify any conversion or distortions only a translation. The coodinate axes of the _IfcGeometricRepresentationContext_ can rotate and set a new basis coordinate system.
```

Comment:
```suggestion
A translation operation specifies an offset in the coordinate reference system. It does not specify any transformation (conversion or distortions) only a translation. The coordinate axes of the _IfcGeometricRepresentationContext_ can rotate and set a new basis coordinate system.
```

Reply:
I need to think about this definition a bit more; I'm not yet happy with it.

Reply:
I propose to use `rigid transformation` for this; need to think about the exact wording, though.